gravitee-apim-gateway/gravitee-apim-gateway-reactor/src/main/java/io/gravitee/gateway/jupiter/reactor/processor/PlatformProcessorChainFactory.java

```diff
@@ -88,7 +87,7 @@ private void initPreProcessorChain() {
         }
 
         preProcessorList.add(transactionHandlerFactory.create());
-        preProcessorChain = new ProcessorChain("processor-chain-post-platform", preProcessorList);
+        preProcessorChain = new ProcessorChain("processor-chain-pre-platform", preProcessorList);
```

Comment:
I did not see other usage of it, so I don't know the purpose of `"processor-chain-pre-platform"`, but it looks like something that will not change.

maybe be it can be extracted as a constant somewhere, as for attributes ?

Reply:
IMHO it's a single usage that does not really requires a constant for now.